src/sponge/truncated.rs

```diff
@@ -8,6 +8,7 @@
 
 use crate::sponge;
 use dusk_bls12_381::BlsScalar;
+use dusk_jubjub::JubJubScalar;
```

Comment:
Why do you specifically import `JubJubScalar` here? afaik it should be included in `dusk_plonk::prelude::*` which we need for other types anyway.